lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1415,5 +1416,14 @@ def api_client
       end
       @client
     end
+
+    # Encode by UTF-8 with invalid characters replaced by spaces.
+    def coerce_to_utf8_encode(input)
```

Comment:
Looking at how the old JSON approach handled invalid UTF-8 characters at the code Igor pointed out: https://cs.corp.google.com/piper///depot/google3/net/proto2/util/converter/internal/json_stream_parser.cc?l=152&rcl=133771901, it's throwing away invalid characters and replacing them with spaces. So here I'm trying to mock the behavior. Note that in that code base, we are also using a boolean flag to control whether to allow non UTF-8 encoded input (aka, whether to replace them or error out). Do we need a similar flag here?